# Map drives numbers to lcec slaves
# names will determine the name of the component pins
hal_402_device_mgr:
    # update rate of publisher in Hz
    update_rate: 10
    drives:
        name: "drive"
        instances: [1, 2, 3, 4, 5, 6]
    slaves:
        name: "lcec.0"
        wait_on_pinname: "warning"
        wait_timeout: 20
        instances: [0, 1, 2, 3, 4, 5]
    # additional signals can be connected.
    signals:
        - ["hal_402_mgr.drive_1.operation-enabled", "joint1_jplan.0.enable"]
        - ["hal_402_mgr.drive_2.operation-enabled", "joint2_jplan.0.enable"]
        - ["hal_402_mgr.drive_3.operation-enabled", "joint3_jplan.0.enable"]
        - ["hal_402_mgr.drive_4.operation-enabled", "joint4_jplan.0.enable"]
        - ["hal_402_mgr.drive_5.operation-enabled", "joint5_jplan.0.enable"]
        - ["hal_402_mgr.drive_6.operation-enabled", "joint6_jplan.0.enable"]
